src/store/actions/authActions.js

```diff
@@ -7,6 +7,9 @@ export const signIn = (credentials) => {
       .auth()
       .signInWithEmailAndPassword(credentials.email, credentials.password)
       .then(() => {
+        // Go online --> Writes "online" into user status
```

Comment:
This is not the norm.
```suggestion
        // Go online. This writes "online" into user status.
```